### PR TITLE
229 create capital project schema for open api

### PR DESCRIPTION
**Pull request by pratishta**
Closes https://github.com/NYCPlanning/ae-zoning-api/issues/229

### CODE REVIEW

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -470,7 +470,44 @@ components:
...
         - "Lump Sum"
         - "ITT, Vehicles and Equipment"
-      descirption: The type of Capital Project.
+      description: The type of Capital Project.
+    CapitalProject:
+      type: object
+      properties:
+        id:
+          type: string
+          description: A string of variable length, composite of managing code and project ID
```
Comment (TangoYankee) [inline openapi/openapi.yaml:479]: There is no single field that represents the whole of the composite key. The composite key exists as a combination of the "id" and "managingCode". I recognize the ambiguity of calling it 'id'; it makes it read like a thing that should be unique. Unfortunately, I don't think we're going find a perfect name. We just need to remember that `id` is not the composite key. It is one of the two fields that compromises the composite key. 

```suggestion
          description: The id for the project, which combines with the managing code to make a unique id
```

**Review by TangoYankee — CHANGES_REQUESTED** [review]

File: openapi/openapi.yaml [hunk 2]
```diff
@@ -470,7 +470,44 @@ components:
         - "Fixed Asset"
         - "Lump Sum"
         - "ITT, Vehicles and Equipment"
-      descirption: The type of Capital Project.
+      description: The type of Capital Project.
```
Comment (TangoYankee) [inline openapi/openapi.yaml:473]: nice catch

File: openapi/openapi.yaml (hunk 1 above)
Comment (pratishta) [reply openapi/openapi.yaml:479]: Wait so the example of this is `HWPEDSF5`-- is that the ID that gets combined with a separate managing code? and `HWPEDSF` big is just part of the project ID?

Comment (pratishta) [reply openapi/openapi.yaml:479]: Ok yes because managing codes are numbers. Okay cool

**Review by TangoYankee — DISMISSED** [review]

**Review by TangoYankee — APPROVED** [review]

**Pull request merged**